rest_framework_simplejwt/authentication.py

```diff
@@ -23,6 +23,10 @@ class JWTAuthentication(authentication.BaseAuthentication):
     """
     www_authenticate_realm = 'api'
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.user_model = get_user_model()
```

Comment:
IIRC, I take back what I said before. I think this should be used at the module level since it's used below too (didn't see it before). Plus, it'll stay with consistency with the other files.

Reply:
@Andrew-Chen-Wang actually this is the bug!!, this module is being imported by rest framework before the user app or model is being ready so we use it at the module level

Reply:
Interesting, I thought it was because of the continuous relative imports, but that does make sense!